modules/web/src/app/core/components/changelog/template.html

```diff
@@ -15,15 +15,7 @@
 -->
 <div fxFlex
      fxLayout="column">
-  <div class="header-title">
-    <div>What's New</div>
-    <div fxFlex></div>
-    <div>
-      <i class="km-icon-close km-pointer"
-         aria-hidden="true"
-         mat-dialog-close></i>
-    </div>
-  </div>
+  <km-dialog-title>What's New</km-dialog-title>
```

Comment:
maybe we need to remove the `.header-title` from the style file since i think we don't use the class any where

Reply:
You're right, I've removed it now.